pyqg/model.py

```diff
@@ -394,129 +394,6 @@ def stability_analysis(self,bottom_friction=False):
 
         return omega, phi
 
-    @property
-    def _config(self):
-        r"""Returns a copy of the set of parameters that can be used to
-        initialize the model in the same way (required for computing subgrid
-        forcing automatically).
-
-        Note that this may need to be overridden in subclasses if parameter
-        names do not match attributes.
-        """
-        res = {}
-        sig1 = inspect.signature(self.__class__.__init__)
-        sig2 = inspect.signature(Model.__init__)
-        for sig in [sig1, sig2]:
-            for param in sig.parameters:
-                if hasattr(self, param):
-                    res[param] = getattr(self, param)
-        return res
-
-    def _reinitialized(self, **kw):
-        r"""Return a new model with the same parameters, except for overrides
-        given in keyword arguments.
-
-        Returns
-        -------
-        model : Model
-            Newly initialized model with same or overridden parameters.
-        """
-        config = self._config
-        config.update(kw)
-        return self.__class__(**config)
-
-    def subgrid_forcing(self,
```

Comment:
Instead of just removing, I would do this
```python
    def subgrid_forcing(self, **kwargs):
        raise ValueError(
            "This method was found to be buggy and removed for now. "
            "It will be re-added in a future release. "
            "See https://github.com/pyqg/pyqg/pull/308 for details. "
        )
```